pyInterface/pyUtils/rootConverters_py.cc

```diff
@@ -172,7 +173,8 @@ void rpwa::py::exportRootConverters() {
 	);
 
 	rpwa::py::setBranchAddress<rpwa::fitResult*>(0, 0, "");
-	rpwa::py::setBranchAddress<rpwa::ampIntegralMatrix*>(0, 0, "");
-	rpwa::py::branch<rpwa::fitResult*>(0, 0, "");
+	rpwa::py::branch<rpwa::fitResult*>(0, 0, "", 32000, 99);
+	rpwa::py::setBranchAddress<rpwa::amplitudeTreeLeaf*>(0, 0, "");
+	rpwa::py::branch<rpwa::amplitudeTreeLeaf*>(0, 0, "", 32000, 99);
```

Comment:
Wouldn't it be better to explicitly instantiate the functions instead of trying to call them here?

```
diff --git a/pyInterface/pyUtils/rootConverters_py.cc b/pyInterface/pyUtils/rootConverters_py.cc
index 5d2cddb..05fbbf0 100644
--- a/pyInterface/pyUtils/rootConverters_py.cc
+++ b/pyInterface/pyUtils/rootConverters_py.cc
@@ -46,6 +46,10 @@ int rpwa::py::setBranchAddress(T objectPtr, PyObject* pyTree, const std::string&
                return tree->SetBranchAddress(name.c_str(), pointerMap[objectPtr]);
 }

+// explicit template instantiation
+template int rpwa::py::setBranchAddress<rpwa::fitResult*>(rpwa::fitResult* objectPtr, PyObject* pyTree, const std::string& name);
+template int rpwa::py::setBranchAddress<rpwa::amplitudeTreeLeaf*>(rpwa::amplitudeTreeLeaf* objectPtr, PyObject* pyTree, const std::string& name);
+
 template<typename T>
 bool rpwa::py::branch(T objectPtr, PyObject* pyTree, const std::string& name, int bufsize, int splitlevel)
 {
@@ -67,6 +71,10 @@ bool rpwa::py::branch(T objectPtr, PyObject* pyTree, const std::string& name, in
                }
 }

+// explicit template instantiation
+template bool rpwa::py::branch<rpwa::fitResult*>(rpwa::fitResult* objectPtr, PyObject* pyTree, const std::string& name, int bufsize, int splitlevel);
+template bool rpwa::py::branch<rpwa::amplitudeTreeLeaf*>(rpwa::amplitudeTreeLeaf* objectPtr, PyObject* pyTree, const std::string& name, int bufsize, int splitlevel);
+
 namespace {

        template<typename T>
@@ -172,9 +180,4 @@ void rpwa::py::exportRootConverters() {
                , bp::return_internal_reference<1>()
        );

-       rpwa::py::setBranchAddress<rpwa::fitResult*>(0, 0, "");
-       rpwa::py::branch<rpwa::fitResult*>(0, 0, "", 32000, 99);
-       rpwa::py::setBranchAddress<rpwa::amplitudeTreeLeaf*>(0, 0, "");
-       rpwa::py::branch<rpwa::amplitudeTreeLeaf*>(0, 0, "", 32000, 99);
-
 }
```